etc/scripts/get-stack.sh

```diff
@@ -18,6 +18,12 @@
 # Make pull requests at:
 # https://github.com/commercialhaskell/stack/blob/master/etc/scripts/get-stack.sh
 #
+# Note that this script will ask for root access using `sudo` in order to use
+# your platform's package manager to install dependencies and to install to
+# `/usr/local/bin`.  If you prefer more control, follow the manual
```

Comment:
Where `stack upgrade` (which is recommended way to upgrade `stack`) will put new `stack` executable?

> You can install stack by copying it anywhere on your PATH environment variable. We recommend installing in the same directory where stack itself will install executables (that way stack is able to upgrade itself!). 

https://docs.haskellstack.org/en/stable/install_and_upgrade/#path

Reply:
Looks like the docs you quoted are out of date.  `stack upgrade` will overwrite its own executable, wherever it is, with the new version (prompting the user whether it's ok to use `sudo` if that's necessary).

Reply:
I've pushed a commit that removes the out-of-date info.